src/backend/defs/rest-api/paths/import.yaml

```diff
@@ -0,0 +1,11 @@
+contacts_import:
+  post:
+    consumes:
+    - text/vcard
+    - text/vcf
+    - text/x-vcard
+    responses:
+      '200':
+        description: Import successful
+      '400':
+        description: Syntax error
```

Comment:
Different content-type permit to import differents format.
I think a generic route is not so bad